build/Toolset/Toolset.csproj

```diff
@@ -1,13 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\Targets\SettingsSdk.props" />
   <PropertyGroup>
     <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <ProjectGuid>{57557F32-0635-421A-BCF8-549AED50F764}</ProjectGuid>
```

Comment:
If it's completely unnecessary then please add a check to BuildBoss to prevent it from coming back. Follow the pattern for ProjectTypeGuids in this section

https://github.com/dotnet/roslyn/blob/master/src/Tools/BuildBoss/ProjectCheckerUtil.cs#L57